forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -612,15 +612,17 @@ public boolean changeCardState(final String mode, final String customState, fina
             // run valid Replacement here
             getGame().getReplacementHandler().run(ReplacementType.Transform, AbilityKey.mapFromAffected(this));
 
-            // do the Transform trigger here, it can also happen if the resulting state doesn't change
-            // Clear old dfc trigger from the trigger handler
-            getGame().getTriggerHandler().clearActiveTriggers(this, null);
-            getGame().getTriggerHandler().registerActiveTrigger(this, false);
+            if (!mode.equals("Convert")) {
+                // do the Transform trigger here, it can also happen if the resulting state doesn't change
+                // Clear old dfc trigger from the trigger handler
+                getGame().getTriggerHandler().registerActiveTrigger(this, false);
 
-            if (cause == null || !cause.hasParam("ETB")) {
-                final Map<AbilityKey, Object> runParams = AbilityKey.mapFromCard(this);
-                getGame().getTriggerHandler().runTrigger(TriggerType.Transformed, runParams, false);
+                if (cause == null || !cause.hasParam("ETB")) {
+                    final Map<AbilityKey, Object> runParams = AbilityKey.mapFromCard(this);
+                    getGame().getTriggerHandler().runTrigger(TriggerType.Transformed, runParams, false);
+                }
             }
+            getGame().getTriggerHandler().clearActiveTriggers(this, null);
```

Comment:
Any specific reason you moved the `clearActiveTriggers` below? I think the order matters and might lead to side effects otherwise

Reply:
Oops, no, that was just typographical